### ISSUE TITLE
Hapcut2 parser -> vcf

### ISSUE THREAD

**Issue by fritzsedlazeck** [fulcrumgenomics/fgbio#390]
Hi,
I did notice that unphased SNPs are reported as phased 0|1 SNPs in the vcf.
Here is an example from my hapcut2 output:
> 352     1       0       chr1    199492  G       A       0/1:5:13:18:.:63,0,322  0       .       37.89
> 353     -       -       chr1    199493  A       G       0/1:4:14:18:.:34,0,350  1       .       3.01
> 354     1       0       chr1    199589  G       A       0/1:2:2:4:.:18,0,48     0       .       19.00
> 355     -       -       chr1    200930  G       A       0/1:4:20:24:.:16,0,504  1       .       3.01
> 356     1       0       chr1    201549  G       A       0/1:3:11:14:.:17,0,276  0       .       37.89

and here are the two SNPs reported with '-':
>chr1    199493  .       A       G       10      single_strand   P=0.904197      GT:DP:NE:PL:PR:PS:RR:SE:VR      0|1:18:3.01:34,0,350:1:198894:14:.:4
>chr1    200930  .       G       A       5       single_strand   P=0.662625      GT:DP:NE:PL:PR:PS:RR:SE:VR      0|1:24:3.01:16,0,504:1:198894:20:.:4

I think that is a bug since you don't have the information available, right?

Thanks
Fritz

**Comment by nh13** [comment]
If you look at field 9 of your HapCut2 output, you'll see that those to SNPs are "pruned", since you almost certainly used the `--discrete_pruning 1` option (please post your full command line, inputs, etc, for the future).  The output VCF for those variants has the value `1` for the `PR` genotype format field, indicating they were pruned.  Since the meaning of "discrete pruned" means that they could be equally likely from either haplotype, I have chosen to include them in the phase set (phase block) to be consistent with the SNPs in the same block, but annotate them as pruned (via the `PR` genotype format field).  You can certainly filter them out using this tag.

Would you have expected those variants to also be set as unphased and not part of a phase set (`PS` genotype format tag)?  I certainly could see what it would take to add such a command line option.

**Comment by fritzsedlazeck** [comment]
Hi, thanks for the fast reply. 
1. I did not use this option. I ran hapcut2 as suggested on their github. 
2. I had this as a question submitted to their github since I was not able to find the meaning of -. 
They responded that at this examples the quality was to low to assign it to a haplotype. If that is true I would have expected that this entry is ignored. However, if you say that is not the case than this clears it up for me .

Thanks
Fritz
